🔍️(SEO) add sitemap to robots

igobranco opened the pull request:
Add sitemap.xml to robots.txt, so it's no longer need to register the sitemap.xml URL to each crawler administration panel

sampaccoud commented on sandbox/urls.py:
```diff
@@ -26,6 +27,16 @@
+    # crawler administration panel
+    path(
+        r"robots.txt",
+        lambda request: HttpResponse(
```
Can we make it a real template for clarity?
Also note that this `urls.py` file is just for sandbox purposes so it will just serve as an example but each site will have its own `urls.py` file it its project.

igobranco replied:
Ok, no problem. Done.

sampaccoud commented on sandbox/urls.py:
```diff
@@ -26,6 +26,14 @@
+    # Add sitemap.xml URL to the robots.txt so we don't need to register sitemap.xml from each
+    # crawler administration panel
+    path(
+        r"robots.txt",
```
Actually this is not a regex:
```suggestion
        "robots.txt",
```

sampaccoud commented on sandbox/urls.py:
```diff
@@ -26,6 +26,14 @@
 
```
Nearly there: don't forget to modify the `CHANGELOG.md` file

igobranco commented:
@sampaccoud I've added a simple test that check the presence of the sitemap rule; fixed the urls.py by removing the `r` and added a new changelog entry.

sampaccoud commented:
@igobranco Make sure to squash your commits :sweat_smile:

igobranco commented:
> @igobranco Make sure to squash your commits

Done. Sorry for this, I'm not used to send pull request...

sampaccoud commented:
@igobranco The CI didn't trigger. Can you try force pushing the branch again?

igobranco commented:
@sampaccoud  I don't know why it's failing on circle ci. On my machine the test `test_robots_sitemap` works fine.

sampaccoud commented:
> @sampaccoud I don't know why it's failing on circle ci. On my machine the test `test_robots_sitemap` works fine.

I think you just forgot to commit the new template file.
